plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveInMemoryMetastore.java

```diff
@@ -61,4 +61,10 @@ public void testHideDeltaLakeTables()
     {
         throw new SkipException("not supported");
     }
+
+    @Override
+    public void testDisallowQueryingOfIcebergTables()
+    {
+        throw new SkipException("not supported");
```

Comment:
why disabled?

add `assertThatThrownBy(super::` ...

Reply:
While trying to run the test I receive

```

java.lang.IllegalArgumentException: Table directory does not exist

	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
	at io.trino.plugin.hive.metastore.thrift.InMemoryThriftMetastore.createTable(InMemoryThriftMetastore.java:190)
	at io.trino.plugin.hive.metastore.thrift.BridgingHiveMetastore.createTable(BridgingHiveMetastore.java:205)
	at io.trino.plugin.hive.AbstractTestHive.testDisallowQueryingOfIcebergTables(AbstractTestHive.java:2978)
	at io.trino.plugin.hive.TestHiveInMemoryMetastore.testDisallowQueryingOfIcebergTables(TestHiveInMemoryMetastore.java:68)
```

Reply:
Which directory? 
This sounds like a test setup problem. Does it mean `TestHiveInMemoryMetastore` cannot run a test that uses `createTable`?

Reply:
I updated the comment to contain the missing directory path.

Note that the similar test `testHideDeltaLakeTables` is also containing a `not supported` skip exception.

https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveInMemoryMetastore.java#L59-L63